presto-main/etc/config.properties

```diff
@@ -40,6 +40,7 @@ plugin.bundles=\
   ../presto-tpch/pom.xml, \
   ../presto-local-file/pom.xml, \
   ../presto-mysql/pom.xml,\
+  ../presto-memsql/pom.xml,\
```

Comment:
There is no need to add it here.
This file is used for development only and I don't expect memsql to be used in local dev very often.

Reply:
Let's keep it here (per current practice), and we can revisit this topic in separate PR.